File: src/main/java/com/Khata/Khata/Service/UserService.java
package com.Khata.Khata.Service;

import com.Khata.Khata.Entity.User;
import com.Khata.Khata.Repository.UserRepository;
import org.springframework.stereotype.Service;

@Service
public class UserService
{
    private UserRepository userRepository;

    UserService(UserRepository userRepository)
    {
        this.userRepository = userRepository;
    }

    public User addUser(User user)
    {
        User savedUser = userRepository.save(user);
        return savedUser;
    }
    
    public User getUserById(Integer id)
    {
        return userRepository.findById(id).get();
    }
}

File: classtracker/src/main/kotlin/com/classtrackeer/classtracker/Service/UserService.kt
package com.classtrackeer.classtracker.Service

import com.classtrackeer.classtracker.Model.User
import com.classtrackeer.classtracker.Repository.UserRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class UserService (@Autowired val userRepository : UserRepository){

    fun addUser(user : User) : User {
        return userRepository.save(user)
    }
}